ivy/functional/frontends/torch/tensor.py

```diff
@@ -988,6 +988,24 @@ def count_nonzero(self, dim):
     def exp(self):
         return torch_frontend.exp(self)
 
+    @with_unsupported_dtypes(
+        {
+            "1.11.0 and below": (
+                "int8",
+                "int16",
+                "int32",
+                "int64",
+                "uint8",
+                "bool",
+                "float16",
+            )
+        },
+        "torch",
+    )  # noqa
```

Comment:
I think it is necessary, because one the tests will fail (Workaround for that is using valid in available_dtypes in tests) and two it is an inplace operation and can't store float result into int. I think it would be difficult to summarize it here. It would be better if I share the discord conversation link: https://discord.com/channels/799879767196958751/933380895358988288/1101156126218801233

Reply:
I have added # fmt: off in the code and it is working, but what are your suggestions on this?
We can remove some of the dtypes or remove this line completely for the time being. When the issue with test identifier is fixed, we can add `@with_unsupported_dtypes`.